ballsdex/core/models.py

```diff
@@ -423,6 +426,18 @@ class Player(models.Model):
     def __str__(self) -> str:
         return str(self.discord_id)
 
+    async def is_friend(self, other_player: "Player") -> bool:
+        return await Friendship.filter(
+            (Q(player1=self) & Q(player2=other_player))
+            | (Q(player1=other_player) & Q(player2=self))
+        ).exists()
```

Comment:
```suggestion
        return await Friendship.filter(
            (Q(player1=self) & Q(player2=other_player))
        ).exists()
```

the prev comment should be resolved with this